content/blog/2020-06-03-how-learnr-saved-the-day/index.markdown

```diff
@@ -0,0 +1,126 @@
+---
+title: How learnr Saved the Day
+author: Ezgi Karaesmen
+date: '2020-06-03'
+description: |
+   My attempt at briefly introducing R to a group of teenagers and how learnr saved me from boring students to death.
+slug: learnr-saved-the-day
+photo:
+  url: https://unsplash.com/photos/6yjAC0-OwkA
+  author: Esteban Lopez
+categories:
+  - teach
+tags:
+  - learnr
+---
+
+Instructors and learners alike have had to quickly adapt to virtual education due to the ongoing pandemic. The [learnr](https://rstudio.github.io/learnr/) package has become a popular tool to ease that transition (see Allison Horst's great  [post](https://education.rstudio.com/blog/2020/05/learnr-for-remote/)).
+
+Before our lives was completely changed by the pandemic, however; I was fortunate enough to conduct a *real life* introductory R session with a group of **teenagers**. I was able to create the materials for this session, [Understanding Injury Data with R](https://suchestoncampbell-lab.shinyapps.io/SummerCamp2019/), with the learnr package and the source code is available on [GitHub](https://github.com/karaesmen/SummerCamp2019/tree/master/tutorial). Although in person examples of teaching may not be the most relevant topic under these circumstances, I thought that sharing my experience of teaching a younger audience using learnr could offer a unique perspective. 
+
+## Some background
+
+I was asked to teach an introductory R session as a part of the [Data Science for Women Summer Camp](https://tdai.osu.edu/summer-camp/) in July 2019. The free of cost camp was organized by the Translational Data Analytics Institute at The Ohio State University (OSU). Admission was available for young women entering grades 8, 9, or 10 attending local high schools in the metro city area. 
+
+<img src="GroupPic.jpg" width="80%" style="display: block; margin: auto;" />
+
+At the time I was asked to instruct the session, I was still a PhD student at OSU and the co-organizer of R-Ladies Columbus (I am still today). Camp organizers had seen me present several introductory R sessions in local conferences and R-Ladies meetups, and thought I would be a great instructor for **the only hands on coding session** of the entire data science camp. 
+
+Camp organizers asked me to:
+
+1. Show doing data analytics with R   
+2. Expect no statistics background from students   
+3. Use a health related dataset  
+4. Do all of this in 50 minutes
+
+Let me assure you they didn't come to me because they were amazed by my well commented, squeaky clean code, or my vast R/statistics knowledge. They invited me because thought I was very enthusiastic and had a "great energy", which they believed was the missing ingredient for this session in the past. It turns out that hands-on-coding sessions weren't a favorite among students the year before.
```

Comment:
- "well commented, squeaky clean code, or my vast..." reads like three clauses rather than two - took me a moment to parse.
- remove quotes around "great energy" (or just remove "and had a great energy")